include/uxr/agent/message/OutputMessage.hpp

```diff
@@ -64,12 +65,37 @@ inline bool OutputMessage::append_submessage(dds::xrce::SubmessageId submessage_
     return rv;
 }
 
+inline bool OutputMessage::append_raw_payload(dds::xrce::SubmessageId submessage_id,
+                                              const uint8_t* buf,
+                                              size_t len,
+                                              uint8_t flags)
+{
+    bool rv = true;
+    if (append_subheader(submessage_id, flags, len))
+    {
+        try
+        {
+            serializer_.serializeArray(buf, len, fastcdr::Cdr::BIG_ENDIANNESS);
+        }
+        catch(eprosima::fastcdr::exception::NotEnoughMemoryException & /*exception*/)
+        {
+            std::cout << "serialize eprosima::fastcdr::exception::NotEnoughMemoryException" << std::endl;
```

Comment:
`std::cerr` instead? Maybe there are more occurrences and it can be a new task.

Reply:
There were more `cout` which I changed by `cerr`.